packages/teuchos/comm/src/Teuchos_PerformanceMonitorBase.cpp

```diff
@@ -324,7 +324,6 @@ namespace Teuchos {
                            const int myRank,
                            const int left,
                            const int mid,
-                           const Array<std::string>& localNames,
                            Array<std::string>& globalNames,
```

Comment:
Please update the documentation if you plan to remove this parameter.

Reply:
Done. Thanks for catching that!